dask/array/utils.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+from .core import Array
+from ..async import get_sync
+
+def eq(a, b, **kwargs):
```

Comment:
To me, `eq` sounds like it simply returns whether their equivalent. Both `numpy` and `pandas` provide both equivalence checks and equivalence assertition functions (e.g. `np.allclose` and `np.testing.assert_allclose`). Personally I'd prefer the slightly longer `assert_eq`, but it's just a preference.